tests/test_events.py

```diff
@@ -26,11 +26,13 @@ def test_handle_listing_processing(
 ):
     config = {}
     listing['status'] = status
-    client_mocker = client_mocker_factory(base_url=connect_client.endpoint)
 
-    client_mocker.marketplaces.filter(R().id.in_([marketplace['id']])).limit(1).mock(
-        return_value=[marketplace],
-    )
+    if status == 'listed':
```

Comment:
[COD-C] Here instead of an `if` statement we must have separate tests for the ones who makes calls with connect client for listed / installed and the others who don't like `test_unlisted_do_nothing` / `test_uninstalled_do_nothing` , and remove the marker of parametrize. But we can leave this as an improvement, wdyt?